### PR TITLE
[wip] generation of kube-scheduler api

### PR DESCRIPTION
**Pull request by kbhawkey**
Initial generation of kube-scheduler api reference.

### CODE REVIEW

File: content/en/docs/reference/scheduling/kubescheduler-api.md [hunk 1]
```diff
@@ -0,0 +1,2238 @@
...
+weight: 30
+---
+
+<p>Packages:</p>
+<ul>
+<li>
+<a href="#kubescheduler.config.k8s.io%2fv1">kubescheduler.config.k8s.io/v1</a>
+</li>
+<li>
+<a href="#kubescheduler.config.k8s.io%2fv1alpha1">kubescheduler.config.k8s.io/v1alpha1</a>
```
Comment (alculquicondor) [inline content/en/docs/reference/scheduling/kubescheduler-api.md]: We are removing this API for 1.19 kubernetes/kubernetes#89260

But if it's complicated to whitelist, we can leave the removal for later once the code is removed.

File: content/en/docs/reference/scheduling/kubescheduler-api.md [hunk 2]
```diff
@@ -0,0 +1,2238 @@
+---
```
Comment (alculquicondor) [inline content/en/docs/reference/scheduling/kubescheduler-api.md]: Shouldn't this go in `static/docs/reference/generated`?

**Comment by kbhawkey** [comment]
Preview:
https://deploy-preview-19903--kubernetes-io-master-staging.netlify.com/docs/reference/scheduling/kubescheduler-api/

**Comment by kbhawkey** [comment]
This is a TEST.

**Comment by tengqm** [comment]
Looks good to me.

**Comment by kbhawkey** [comment]
Closing as there appears to be less interest (pending the adoption of the new theme).
/close

**Pull request closed without merging**